app/templates/events/view/tickets/index.hbs

```diff
@@ -0,0 +1,98 @@
+<div class="{{if device.isMobile "ui stackable three horizontal statistics" "ui three statistics"}}">
+  <div class="statistic"><div class="value">4343</div><div class="label">{{t 'Orders'}}</div></div>
+  <div class="statistic"><div class="value">0.00</div><div class="label">{{t 'Sales(US $)'}}</div></div>
+  <div class="statistic"><div class="value">1</div><div class="label">{{t 'Tickets sold'}}</div></div>
+</div>
+<br>
+<div class="row">
+  <div><span class="ui header">{{t 'Order Summary'}}&nbsp;</span><span class="sub header">{{t '-By order status'}}</span></div>
```

Comment:
Please format all this properly. Don't put all on a single line. The sub header should be inside a div with class `ui header`. Please see semantic UI documentation properly.

```hbs
<div class="ui header">
    {{t 'Order Summary'}}
    <div class="sub header">
        {{t 'By order status'}}
    </div>
</div>
```